### ISSUE TITLE
GenesisMint tx type does not include initiator  (Version: 2024.1.25-release+738)

### ISSUE THREAD

**Issue by shortthefomo** [Xahau/xahaud#280]
## Summary
look up / watch live tx the transaction who initiates the balance claim is not the owner.

## Motivation
owner of tx should be simple to look up

## Solution
add the owner of the balance claim to the result

## Paths Not Taken
blamed daniel

**Comment by zgrguric** [comment]
EmitParentTx initiator is The Initiator.
This transaction piggybacks on Hooks Ammendment system. Eg it is direct result of execution of hook:

https://xahau.xrplwin.com/hook/610F33B8EBF7EC795F822A454FB852156AEFE50BE0CB8326338A81CD74801864/C000000300005359/txs

It is possible to lookup parent TX.

Even “cheaper” lookup who is initiator is one with most XAH awarded, as per hook rules other accounts get calculated percentage only.

My opinion is that nothing needs to be changed.

**Comment by shortthefomo** [comment]
Maybe we should have this as a common function in a tooling lib then, assume this same case is going to exist on other hooks. Apps etc, want to look up who initiated the transaction via the hook?

I only filed this with the GenesisMint tx type as that's where I looked first.


I would however say that ability to know the initiator of the tx that occurred via the hook will be a pretty common request.
